src/test/shell/bazel/path_mapping_test.sh

```diff
@@ -0,0 +1,290 @@
+#!/bin/bash
+#
+# Copyright 2023 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Tests path mapping support of Bazel's executors.
```

Comment:
To be clear, this is for Bazel-specific behavior while `config_stripped_outputs_test` is generic?

Reply:
Yes, I don't know whether this would also work for Blaze. It's based on Bazel's flag defaults and also makes some assumptions about e.g. the total number of actions in a build.